services/frontend/src/components/ProjectInfoDialog/ProjectInfoDialog.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import {useInput} from "utils/hooks";
+import {Button, Dialog, DialogActions, DialogContent, DialogTitle} from "@material-ui/core";
+import {DetailsForm} from "components/";
+import "./ProjectInfoDialog.scss";
+
+const ProjectInfoDialog = ({project, closeDialog, open}) => {
+    const formFieldData = {
+        "nameInput": {
+            ...useInput(project.name),
+            id: "name",
+            label: "Name",
+            autoFocus: true
+        },
+        "descriptionInput": {
+            ...useInput(project.description),
+            id: "description",
+            label: "Description",
+            multiline: true
+        },
+    };
+
+    return (
+        <Dialog className="project-info-dialog"
```

Comment:
Just like how you refactored `DetailsForm` to be a shareable component, you could probably refactor a good amount of this dialog since a lot of it is shared between the `ProjectInfoDialog` and `PersonalDetailsDialog`.

Reply:
Alright, I moved what I can to a separate component and moved some of the data up to a component in ProfileLayout to keep it somewhat clean and work with CDHSH-89.